src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsFandom.java

```diff
@@ -0,0 +1,100 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.fandom.AdsFandomTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+
+public class TestAdsFandom extends AdsFandomTestTemplate {
+
+  private void getJquery() {
```

Comment:
Why jquery is needed? Correct me if I'm wrong but it's needed only to change the opacity of ad slots to make screenshot comparision. So instead of adding it to every page we can change the opacity with pure JS.

https://github.com/Wikia/selenium-tests/blob/7d758ff4f55f5e6e8be0d6140c8079cce5ee5665/src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/helpers/AdsComparison.java#L48
`document.querySelector(arguments[0]).style.opacity = arguments[1];`